### ISSUE TITLE
Question on loss function and evaluation

### ISSUE THREAD

**Issue by yaohuax** [rohanchandra30/Spectral-Trajectory-and-Behavior-Prediction#44]
Hi, Thank you for sharing your work. When i read the code, I got some questions which confused me.

1. When you train stream1, you compute the log likelihood as follows.
```
    y_1 = y[:,0]
    # y_1 = (y_1-torch.mean(y_1))/y_1.max()
    y_2 = y[:,1]
    # y_2 = (y_2 - torch.mean(y_2))/y_2.max()
    mu_1 = torch.mean(y_1) + (y_1 -torch.mean(mu_1))
    # mu_1 = torch.mean(y_1) + (y_1 -torch.mean(mu_1)) * (torch.std(y_1)/torch.std(mu_1))
    mu_2 = torch.mean(y_2) + (y_2 -torch.mean(mu_2))
    # mu_2 = torch.mean(y_2) + (y_2 -torch.mean(mu_2)) * ((torch.std(y_2))/(torch.std(mu_2)))
    z = ( (y_1 - mu_1)**2/(log_sigma_1**2) + ((y_2 - mu_2)**2/(log_sigma_2**2)) - 2*rho*(y_1-mu_1)*(y_2-mu_2)/((log_sigma_1 *log_sigma_2)) )
```
   When you compute the z, ```(y_1 - mu_1)``` should always be ```(torch.mean(mu_1) - torch.mean(y_1))```, which means 
   you get all same loss for a batch of data at timestamp t. 

2. When evaluating the model accuracy, you normalize the ground truth to compare with the output from decoder. As far as i understand, the output from decoder is not global coordinates which can be used directly. So how can I transform the output to coordinates frame that the ground truth used?

Looking forward to your reply.

**Comment by Xiejc97** [comment]
Hello! 

I want  to ask if you can run the comparison_method/traphic_sconv method successfully? For the traphic_sconv/main.py, every time I run it, it will report an error: can't find ../resources/data/XXX/train/trainSet0-traj.npy and trainSet0-track.npy. But in the main.py, I can't find the related code to generate the traj and track file.

Hope you can help me explain the problem, thank you very much!

**Comment by yaohuax** [comment]
> Hello!
> 
> I want to ask if you can run the comparison_method/traphic_sconv method successfully? For the traphic_sconv/main.py, every time I run it, it will report an error: can't find ../resources/data/XXX/train/trainSet0-traj.npy and trainSet0-track.npy. But in the main.py, I can't find the related code to generate the traj and track file.
> 
> Hope you can help me explain the problem, thank you very much!

Sorry, I gave up on this model.

**Comment by Xiejc97** [comment]
So you did not run through the traphic_sconv method correctly，right? Do you think the relevant code for generating traj is missing here? This problem has confuseded me for a long time. If you know something, hope to help me！
